client/tx_generator.go

```diff
@@ -25,7 +25,7 @@ type (
 	// signatures, and provide canonical bytes to sign over. The transaction must
 	// also know how to encode itself.
 	TxBuilder interface {
-		GetTx() sdk.Tx
+		GetTx() signing.SigFeeMemoTx
```

Comment:
Can we think of a better name for this? What exactly is the intention of this interface?

Reply:
It’s what is actually supported by both the proto buf and amino tx. Maybe it should simply be `Tx` or `StdTx` but that would be confusing for other reasons. Maybe `FullTx`? I’m pretty stumped on names

Reply:
Any ideas? Otherwise maybe we change this in another PR?

Reply:
Added a TODO item to #6213 for this.